backend-app/routes/index.ts

```diff
@@ -0,0 +1,21 @@
+import { Router } from 'express';
+import userRoutes from './users/user_route';
+import adminRoutes from './users/admin_route';
+import superAdminRoutes from './users/super_admin_route';
+import authRoutes from './auth_routes';
+import githubRoutes from './github_routes';
+import authController from '../controllers/auth_controllers/auth_controller';
+
+const router = Router();
+
+// public routes
+authRoutes(router);
+
+router.use(authController.protect);
+
+// protected routes
+userRoutes(router);
+adminRoutes(router);
+superAdminRoutes(router);
+githubRoutes(router);
+module.exports = router;
```

Comment:
change this to : export default router

Reply:
done! https://github.com/ISIL-ESTE/Student-Workflow-Organizer/pull/132/commits/7bb56da52435a36c847ecc29a499963b12d3a698